File: DayTwo/Java/TaskFour.java
import java.util.Arrays;
	public class TaskFour{
		public static void main(String[] args){
			int[] array = {1, 2, 3, 4, 5, 6, 7};
			int[] result = answer(array);
			System.out.print(Arrays.toString(result));
		}

		public static int[] answer(int[] number){
			int[] even = new int[number.length];
			for(int counter = 0; counter < number.length; counter++){
				if(counter % 2 == 0){
					even[counter] = number[counter];
				}
			}	
			return even;
		}

	}

File: DayTwo/JavaScript/TaskFour.js
const prompt = require("prompt-sync")();
let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9];
for(let counter = 0; counter < numbers.length; counter++){
	let even = numbers[0];
	if(counter % 2 == 0){
		even[counter] = numbers[counter];
	};

	console.log(even)
}